rollup.config.js

```diff
@@ -115,7 +115,7 @@ if (!buildBundles) {
       { file: pkg.main, format: 'cjs' },
     ],
   };
-  rollupConfigs = [configIIFE, configIIFEMinified];
+  rollupConfigs = [configIIFE, configIIFEMinified, configBundles, configBundlesMinified];
```

Comment:
👍

Reply:
I guess to test the new minified IIFE bundle I removed the other bundles to speedup the generation, and we didn't notice the removal 😈